api/src/main/java/io/strimzi/api/kafka/model/KafkaBridgeClientSpec.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Collections.emptyMap;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@EqualsAndHashCode
+public class KafkaBridgeClientSpec implements UnknownPropertyPreserving, Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    protected Map<String, Object> config = new HashMap<>(0);
+
+    private Map<String, Object> additionalProperties;
+
+    public Map<String, Object> getConfig() {
```

Comment:
A little bit of Javadoc here please as we have for Kafka and Zookeeper related configuration.

Reply:
This class is not used. Its children `KafkaBridgeConsumerSpec` and `KafkaBridgeProducerSpec` with more specific descriptions are used.